From 3348beaf95c84d835aeb249fefa68f839a1623bf Mon Sep 17 00:00:00 2001
From: msieben <github@desiderium.eu>
Date: Thu, 3 Mar 2016 16:24:28 +0100
Subject: [PATCH 2/3] netflix: introduce build flag USE_MANUFSS

The build flag allows compile time selection between a secure store from file or as a download from the backoffice
---
 partner/dpi/reference/DeviceLibNative.cpp | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/partner/dpi/reference/DeviceLibNative.cpp b/partner/dpi/reference/DeviceLibNative.cpp
index f19b1b18..a4bb24d0 100644
--- a/partner/dpi/reference/DeviceLibNative.cpp
+++ b/partner/dpi/reference/DeviceLibNative.cpp
@@ -60,7 +60,7 @@ using namespace std;
 using namespace netflix;
 using namespace netflix::device;
 
-#if 0
+#if USE_MANUFSS
 #include <provision/DRMInfo.h>
 #endif
 
@@ -189,6 +189,7 @@ DeviceLibNative::DeviceLibNative() :
     //If present, read idfile content
     DataBuffer fileContent;
 
+#ifndef USE_MANUFSS
     if (sConfiguration->idFile.empty())
     {
         Log::fatal(TRACE_DPI, "No manufacturing secure store specified. Exiting.");
@@ -196,8 +197,9 @@ DeviceLibNative::DeviceLibNative() :
         Log::warn(TRACE_DPI, "You can also launch netflix using the script \"netflix_manuss.sh\" which will call \"manufss\" before launching netflix");
         exit(0);
     }
+#endif
 
-#if 0
+#if USE_MANUFSS
     // Get manufacturer secure store from back office.
     const unsigned short bufferSize = 8192; // 8 kb
     char netflixId[bufferSize];
-- 
2.17.1

